apps/wing/src/cli.ts

```diff
@@ -3,6 +3,7 @@ import { satisfies } from "compare-versions";
 
 import { optionallyDisplayDisclaimer } from "./analytics/disclaimer";
 import { exportAnalytics } from "./analytics/export";
+import { DEFAULT_PARALLEL_SIZE } from "./commands";
```

Comment:
This import is causing us to eagerly import all the commands (and their dependencies).

This constant could be moved to another file.